src/components/SearchForm/SearchForm.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import "./SearchForm.css";
+
+function SearchForm({ onNewsArticlesSearched }) {
+    const [news, setNews] = useState("");
+
+    const handleChange = (evt) => {
+        setNews(evt.target.value);
+    };
+
+    const handleArticleSubmit = (evt) => {
+        evt.preventDefault();
+        if (!news.trim()) {
+            return;
+        }
+        onNewsArticlesSearched(news);
+    };
+
+  return (
+    <div className="form">
+      <form action="" className="form__search" onSubmit={handleArticleSubmit}>
+        <input
+          type="text"
+          className="form__search-input"
+          placeholder="Enter Topic"
+          required
+          aria-label="News Search"
+          id="search"
+          name="search"
+          value={news}
```

Comment:
Good job that you don't forget about `value` in the inputs if there is `onChange`